### PR TITLE
Add Discord bot links.

### LINKED ISSUE #187: It's not just Slack... IS NOW ALSO DISCORD!
[Discord](https://discordapp.com), a gamer chat and voice software recently added [Animated Emotes](https://twitter.com/discordapp/status/944011021512880128). In less than a few minutes I've found The Legendary Parrot on a lot of Discord Servers.

So perhaps, it's also "The only Discord animated emotes you'll ever need".

PD: In the Twitter link you will find an animation, just admire "shuflewumpus.gif".

### PR DESCRIPTION
**Pull request by jmhobbs**
Would resolve #187.

@Kizzaris and @justalemon, is what I wrote here correct?  I don't really use Discord so I'm not 100% sure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,3 +129,8 @@ installed to run this:
  3. `python upload_parrots.py --team YOUR_TEAM --username YOUR_EMAIL --password YOUR_PASS`
 
  __Notice__: The uploader won't re-write already existing parrots if they match by name.
+
+## Discord via Mr. Parrot
+
+  1. Visit [this invite link](https://discordapp.com/oauth2/authorize?client_id=394830082058747905&permissions=1074006016&scope=bot) to add the bot to your server
+  2. Enjoy!
```
Comment (jmhobbs) [inline README.md]: This is all you have to do, right?

Comment (justalemon) [reply README.md]: The correct phrase would be "Click this link to invite the bot to your server"

Comment (Kizzaris) [reply README.md]: Yeah, what @justalemon is suggesting would fit more.

**Comment by jmhobbs** [comment]
Updated, thanks!

**Pull request merged**